src/audio/copier/copier.c

```diff
@@ -1591,14 +1589,18 @@ static int set_attenuation(struct comp_dev *dev, uint32_t data_offset, const cha
 		comp_err(dev, "attenuation %d is out of range", attenuation);
 		return -EINVAL;
 	}
+	uint32_t __sparse_cache valid_fmt, frame_fmt;
```

Comment:
can we move the declaration to the beginning of this function ?

Reply:
SOF CI

Reply:
Done